packages/test/local-server-tests/src/test/noDeltaStream.spec.ts

```diff
@@ -52,13 +52,17 @@ describe("No Delta Stream", () => {
         return container;
     }
 
-    async function loadContainer(storageOnly: boolean): Promise<IContainer> {
+    async function loadContainer(storageOnly: boolean, track = true): Promise<IContainer> {
+        const service = new LocalDocumentServiceFactory(deltaConnectionServer, { storageOnly });
```

Comment:
1.  should all "storageOnly" container be not tracked.
2. If it is not tracked, it also won't be closed at the end of thetest.   May be the LoaderContainerTracker should be smarter and not require "StorageOnly" containers part of the "synchronized" container for ensuredSyncrhonized?

Reply:
Possibly. Some synchronization is required (as test depends on one of these containers to get current). Changed it to leverage waitContainerToCatchUp(storageOnlyContainer() instead and removed track in favor of !storageOnly